monitoring/loki.yml

```diff
@@ -0,0 +1,7 @@
+apiVersion: 1
+datasources:
+  - name: Loki
+    type: loki
+    access: proxy
+    url: http://loki:3100
+    basicAuth: false
```

Comment:
you are using `promtail.yml` as promtail config, but `loki.yml` as grafana datasource — kinda confusing